### PR TITLE
Update basic hardware requirements for 2019

### PR DESCRIPTION
**Pull request by nmarley**

### CODE REVIEW

**Comment by nmarley** [comment]
@strophy, @thephez Would you be able to take a quick look at this? Does this align more w/basic recommendations in official docs?

File: README.md [hunk 1]
```diff
@@ -12,11 +12,9 @@ Docker image that runs the Dash dashd node in a container for easy deployment.
 Requirements
 ------------
 
-* Physical machine, cloud instance, or VPS that supports Docker (i.e. Vultr, Digital Ocean, KVM or XEN based VMs) running Ubuntu 14.04 or later (*not OpenVZ containers!*)
-* At least 4 GB to store the block chain files
-* At least 1 GB RAM + 2 GB swap file
-
-Recommended and tested on Vultr 1024 MB RAM/320 GB disk instance @ $8/mo.  Vultr also *accepts Bitcoin payments*!  May run on the 512 MB instance, but took *forever* (1+ week) to initialize due to swap and disk thrashing.
+* Physical machine, cloud instance, or VPS that supports Docker (i.e. Vultr, Digital Ocean, KVM or XEN based VMs) running Ubuntu 16.04 or later (*not OpenVZ containers!*)
+* At least 20 GB to store the block chain files
```
Comment (thephez) [inline README.md]: Per https://docs.dash.org/en/stable/masternodes/understanding.html#masternode-requirements, minimum disk is now 40 GB.


  | Minimum | Recommended
-- | -- | --
CPU | 1x 1 GHz | 1x 2 GHz
RAM | 2 GB + 2 GB swap | 4 GB + 2 GB swap
Disk | 40 GB | 60 GB
Network | 400 GB/mth | 1 TB/mth

**Comment by nmarley** [comment]
Thanks for pointing that out. This is for MN minimums, but I wonder how much space is required for a regular (non-MN) node w/o indices enabled. I'll try and start a new one and see how much disk is used, then bump it a little from there.

Probably good to also recommend Ubuntu 18.04 as a minimum for host reqs.

**Comment by nmarley** [comment]
I bumped min disk to 40GB and min Ubuntu to 18.04, but now this depends on #24 being merged first.

**Review by UdjinM6 — APPROVED** [review]
utACK

**Review by thephez — APPROVED** [review]
Better late than never, right? :wink: Not sure how this fell off the radar.

**Comment by nmarley** [comment]
Didn't seem super trivial so I never followed up on these. ¯\\\_(ツ)\_/¯

FWIW, I don't have access to merge these anymore so someone else will have to (cc @UdjinM6).

**Pull request merged**